subcommand/server-acl-init/command.go

```diff
@@ -244,14 +311,52 @@ func (c *Command) getConsulServers(logger hclog.Logger) *apiv1.PodList {
 			}
 			return nil
 		}, logger)
-	return serverPods
+	if err != nil {
+		return nil, err
+	}
+
+	var podAddrs []podAddr
+	for _, pod := range serverPods.Items {
+		var httpPort int32
+		for _, p := range pod.Spec.Containers[0].Ports {
+			if p.Name == "http" {
+				httpPort = p.ContainerPort
```

Comment:
why no option for https port here in addition to http in case for TLS only environments?

Reply:
Hey @filintod, thanks for pointing this out. Currently, consul servers deployed by the helm chart [do not support](https://github.com/hashicorp/consul-helm/blob/24d3115a2f04e8b5058f43b1d67afe6eff20df19/templates/server-statefulset.yaml#L142-L156) https, that's why the code here doesn't have that option. This is definitely something we're looking to add soon.

Reply:
@ishustava I know about the helm chart as I had to go and customize it and this command.go file to support it and looking at how substantial this change is I though it would be good to also add it.  Without this change the helm chart with TLS will not work, there is a PR https://github.com/hashicorp/consul-helm/pull/137 that really does not work without changes in parallel with this consul-k8s tool.  But you can resolve it, maybe adding a bug to add TLS support to keep track and work to be released with helm tls support.

Reply:
Yes, definitely. We didn't have an issue for this, but it was certainly something we were aware of. I've created #154 to track this separately.

Reply:
Thanks for the review Filinto 🙏. Getting TLS working is an important goal for us but I think we'll tackle in a separate PR as our priority is getting these fixes out.

When we start on TLS work we'll tag you.